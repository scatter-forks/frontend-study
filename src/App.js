// import './App.css';

const handleClick = () => {
  alert('LOVE')
}

function App() {
  return (
    <div className="App">
      <h1>Demo</h1>
      <h2 onClick={handleClick}>click will alert LOVE</h2>
    </div>
  );
}

export default App;
