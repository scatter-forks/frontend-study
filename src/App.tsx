import { HashRouter as Router, Route, Switch } from 'react-router-dom'
import { ConfigProvider } from 'antd'

import './App.css'
import routes from './router'

const App = () => {
  return (
    <ConfigProvider>
      <Router>
        <Switch>
          {routes.map((route: any) =>
            <Route key={route.path} path={route.path} exact component={route.component} />)
          }
        </Switch>
      </Router>
    </ConfigProvider>
  )
}

export default App
