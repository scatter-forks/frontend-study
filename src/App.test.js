import App from './App';
import {configure, shallow} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })


describe('App test', () => {
  it('snapshot', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})