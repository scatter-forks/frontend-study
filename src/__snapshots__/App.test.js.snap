// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test snapshot 1`] = `ShallowWrapper {}`;

exports[`demo snapshot 1`] = `ShallowWrapper {}`;
